src/tls/openssl/tls.c

```diff
@@ -1570,10 +1546,9 @@ static int tls_session_update_cache(const struct tls_conn *tc,
 					     session_cmp_handler, &peer));
 	mem_deref(e);
 
-#if (OPENSSL_VERSION_NUMBER >= 0x10101000L) && \
```

Comment:
This check is needed 0x10101000L is openssl 1.1.1

Reply:
I guess also in tls.c:61
```diff
--- b/src/tls/openssl/tls.c
+++ a/src/tls/openssl/tls.c
@@ -61,7 +61,7 @@ struct tls_cert {
 	char *host;
 };
 
-#if defined(TRACE_SSL)
+#if defined(TRACE_SSL) && (OPENSSL_VERSION_NUMBER >= 0x10101000L)
 /**
  * Global flag if key material must be appended to file
  */
```